CMakePresets.json

```diff
@@ -0,0 +1,59 @@
+{
+    "version": 3,
+    "configurePresets": [
+        {
+            "name": "base",
+            "displayName": "base preset",
+            "generator": "Ninja",
+            "binaryDir": "${sourceDir}/build-${presetName}",
+            "hidden": true
+        },
+        {
+            "name": "dev-mold",
+            "displayName": "Build as debug + using mold linker",
+            "cacheVariables": {
+                "CMAKE_BUILD_TYPE": "Debug",
+                "CMAKE_SHARED_LINKER_FLAGS": "-fuse-ld=mold"
+            },
+            "inherits": [
+                "base"
+            ]
+        },
+        {
+            "name": "dev",
```

Comment:
I'm curious: is there a specific reason to prefer dev over debug, when the counter profile is called release? (I think debug and release are more common cmake terms than dev)

Reply:
hi :) just I use by default in kde 'dev' => development
but I will rename as debug :)